main.nf

```diff
@@ -17,7 +17,11 @@ nextflow.enable.dsl = 2
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
+params.fasta         = WorkflowMain.getGenomeAttribute(params, 'fasta')
+params.gtf           = WorkflowMain.getGenomeAttribute(params, 'gtf')
+params.gff           = WorkflowMain.getGenomeAttribute(params, 'gff')
+params.star_index    = WorkflowMain.getGenomeAttribute(params, 'star')
+params.salmon_index  = WorkflowMain.getGenomeAttribute(params, 'salmon')
```

Comment:
I would prefer to have a sorted list, but that's just my personal opinion

Reply:
Fixed in #68